docs/index.md

```diff
@@ -1,58 +1,16 @@
 <link rel="stylesheet" href="/css/blog-home-page.css"></link>
 <script src="/js/blog-home-page.js"></script>
 ---
-* <div class="cBlogThumbnailContainer"><img src="img/blog-images/drone-innovation-technology-57544.jpg"/></div> [Thinking in Ballerina](./posts/thinking-in-ballerina.md)
-* Selvaratnam Uthaiyashankar
-* 14 May 2018
-* Ballerina's syntax, code, and compiler create runtime services and deployment artifacts that are cloud native ready for deployment within your IaaS, orchestrator, or service mesh of choice. The developer experience is designed to maintain flow, where the edit, build, debug cycle is rapid and integrated to your team's lifecycle toolchain.
+* <div class="cBlogThumbnailContainer"><img src="img/blog-images/drone-innovation-technology-57544.jpg"/></div> [Ballerina - A cloud native programming language for integration](./posts/ballerina-a-cloud-native-programming-language.md)
+* Kasun Indrasiri
+* 5 June 2018
+* Ballerina is a cloud-native programming language which aims to fill the gap between integration products and general purpose programming languages by making it easy to write programs that integrate and orchestrate across distributed microservice and endpoints in type-safe and resilient manner.
```

Comment:
Ballerina is a cloud-native programming language that aims to fill the gap between integration products and general purpose programming languages by making it easy to write programs that integrate and orchestrate across distributed microservices and endpoints in a type-safe and resilient manner.